src/layouts/home-assistant.ts

```diff
@@ -68,6 +68,16 @@ export class HomeAssistantAppEl extends HassElement {
     try {
       const { auth, conn } = await window.hassConnection;
       this.initializeHass(auth, conn);
+
+      const curVersion = conn.haVersion;
+
+      // Force a reload if we reconnect to a new version so we load
+      // latest frontend.
+      conn.addEventListener("ready", (conn2) => {
+        if (conn2.haVersion !== curVersion) {
```

Comment:
Shouldn't this be done in `hassReconnected` in the `connection-mixin`?

Reply:
oh reconnected is a good one. 

I didn't want to put it in the connection mixin because it's specific to the frontend, and I wouldn't want to add it into Cast.